app/views/doctors/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Name: <%= @doctor.name %></h1>
+<p><%= "Specialty: #{@doctor.specialty}" %></p>
+<p><%= "University: #{@doctor.university}" %></p>
+<p><%= "Hospital: #{@doctor.hospital.name}" %></p>
+<section id="patients">
+<h2>List of Patients</h2>
+<% @doctor.patients.each do |patient| %>
+  <section id="patient-<%= patient.id %>">
+  <%= patient.name %>
+  <%= button_to "Remove", doctor_patient_path(@doctor.id, patient.id), method: :delete %>
```

Comment:
I don't think the path that you created in the `routes.rb` takes 2 IDs. If you run a `rails routes` you will notice that the produces a path looks something like this: `"/doctor_patients/:id"`, which indicates that only it's only accepting or looking for 1 ID, but you are passing in `@doctor.id` and `patient.id`... and this is why in your controller you are doing something funky with `params[:format]`. 

You are trying to use the route helper here on line 10 as if it is a nested route but didn't set it up as a nested route in your `routes.rb` file.